meeting_minutes/2024/2024-01-31.md

```diff
@@ -0,0 +1,177 @@
+![image](https://user-images.githubusercontent.com/1690898/139102180-5c1e2583-14f1-4f58-ab2b-9e3807ed529c.png)
+
+# Common Security Advisory Framework (CSAF) Technical Committee Working Meeting
+
+- Meeting Date: January 31, 2024
+- Time: 18:00 UTC (19:00 CET, 13:00 EST, 10:00 PST)
+
+## Call to Order and Welcome
+
+Meeting called to order @ 18:04 UTC
+
+## Roll call
+
+All participants recorded their attendance on the OASIS meeting calendar.
+All participants were kindly encouraged to register themselves to optimize the use of the shared time during the meeting in one of two ways:
+
+- Clicking the link with the text "Register my attendance" on the top of the event page.
+- Or directly visiting the per event direct "record my attendance link."
+
+Quorum was reached as 12 of 19 voting members were present.
+
+## Participants
+
+| Given Name | Family Name | Affiliation                                                 | Role                        |
+|:-----------|:------------|:------------------------------------------------------------|:----------------------------|
+| Christoph  | Plutte      | Ericsson                                                    | Member                      |
+| Denny      | Page        | Individual                                                  | Voting Member               |
+| Dina       | Truxius     | Federal Office for Information Security (BSI)               | Voting Member               |
+| Feng       | Cao         | Oracle                                                      | Voting Member               |
+| Jane       | Ginn        | Cyber Threat Intelligence Network, Inc. (CTIN)              | Voting Member               |
+| Justin     | Murphy      | DHS Cybersecurity and Infrastructure Security Agency (CISA) | Voting Member               |
+| Martin     | Prpic       | Red Hat                                                     | Voting Member               |
+| Michael    | Reeder      | Dell                                                        | Voting Member               |
+| Sonny      | van Lingen  | Huawei Technologies Co., Ltd.                               | Voting Member               |
+| Stefan     | Hagen       | Individual                                                  | Voting Member, taking notes |
+| Thomas     | Proell      | Siemens AG                                                  | Voting Member               |
+| Thomas     | Schaffer    | Cisco Systems                                               | Voting Member               |
+| Thomas     | Schmidt     | Federal Office for Information Security (BSI)               | Voting Member               |
+| Tobias     | Limmer      | Siemens AG                                                  | Voting Member               |
+| Vivek      | Nair        | Microsoft                                                   | Member                      |
+
+
+### Observers present
+
+- Tyler Townes, Blackberry Limited
+
```

Comment:
According to what I have seen in the Webex participants list, Jared was also there

@santosomar: Please add him as well.

```suggestion
- Jared Bird, Hewlett Packard Enterprise

```